docs/storing-querying/storage-options/index.md

```diff
@@ -64,6 +64,7 @@ All lake loaders are micro-batching.
 | Lake | Format | Compatibility | Loader application | Status |
 | --- | --- | --- | --- | --- |
 | S3 | Delta    | Amazon Athena, Databricks | [Lake Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/lake-loader/index.md) | Early release |
+| S3 | Iceberg  | Amazon Redshift, Amazon Athena | [Lake Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/lake-loader/index.md) | Early release |
```

Comment:
Amazon Redshift???  We don't normally use the "Amazon" prefix.  But I did it to match Amazon Athena on the preceding row.